### PR TITLE
BaseFilterListService: Relocated saving and loading of active filters

### LINKED ISSUE #1137: Clean up the `BaseListFilterService`
Since #1132 is merged, the `BaseListFilterService` injects the `HistoryService` which is only relevant for meetings. The `HistoryService` (and maybe the `StorageService`, too) should be extracted from the `BaseListFilterService`.

### PR DESCRIPTION
**Pull request by luisa-beerboom**
Closes #1137

### CODE REVIEW

File: client/src/app/site/pages/meetings/services/meeting-active-filters.service.ts
Comment (GabrielInTheWorld) [inline client/src/app/site/pages/meetings/services/meeting-active-filters.service.ts:28]: Good abstraction!

File: client/src/app/site/services/active-filters.service.ts
Comment (GabrielInTheWorld) [inline client/src/app/site/services/active-filters.service.ts:9]: Don't forget the visibility modifier.

File: client/src/app/site/services/active-filters.service.ts
Comment (GabrielInTheWorld) [inline client/src/app/site/services/active-filters.service.ts:12]: Do return this and change the return type to `Promise<void>` -> one can await for the "ready event".

File: client/src/app/site/services/active-filters.service.ts
Comment (GabrielInTheWorld) [inline client/src/app/site/services/active-filters.service.ts:17]:
```suggestion
    public load<V>(storageKey: string): Promise<OsFilter<V>[]> {
        return this.store.get<OsFilter<V>[]>(`filter_` + storageKey);
    }
```

File: client/src/app/site/pages/meetings/services/meeting-active-filters.service.ts
Comment (GabrielInTheWorld) [inline client/src/app/site/pages/meetings/services/meeting-active-filters.service.ts:21]:
```suggestion
    public async save<V>(storageKey: string, filterDefinitions: OsFilter<V>[]): Promise<void> {
        if (!this.historyService.isInHistoryMode()) {
            await this.activeFiltersService.save<V>(storageKey, filterDefinitions);
        }
    }
```

File: client/src/app/site/pages/meetings/services/meeting-active-filters.service.ts
Comment (GabrielInTheWorld) [inline client/src/app/site/pages/meetings/services/meeting-active-filters.service.ts:11]: Don't forget the visibility modifier

**Review by GabrielInTheWorld — APPROVED** [review]
Good abstraction! Some changes have to be made. All in all it is good and after resolving the remarks, it can be merged if @emanuelschuetze approves it, too.

File: client/src/app/site/services/active-filters.service.ts
Comment (luisa-beerboom) [reply client/src/app/site/services/active-filters.service.ts:17]: Why?

**Pull request merged**
